prepare_mkl.sh

```diff
@@ -57,10 +57,10 @@ MXNET_ROOT=`dirname $0`
 USE_MKLML=0
 # NOTE: if you update the following line, please also update the dockerfile at
 # tests/ci_build/Dockerfile.mkl
-VERSION_MATCH=20170210
-ARCHIVE_BASENAME=mklml_lnx_2017.0.2.20170209.tgz
+VERSION_MATCH=20170425
+ARCHIVE_BASENAME=mklml_lnx_2018.0.20170425.tgz
 MKL_CONTENT_DIR=`echo $ARCHIVE_BASENAME | rev | cut -d "." -f 2- | rev`
-MKLURL="https://github.com/dmlc/web-data/raw/master/mxnet/mklml-release/$ARCHIVE_BASENAME"
+MKLURL="https://github.com/01org/mkl-dnn/releases/download/v0.7/$ARCHIVE_BASENAME"
```

Comment:
That version number is not related to the mklml library itself. The library file is residing in mkl-dnn's public repo and the version number is for mkl-dnn.